docker/Dockerfile.contrib

```diff
@@ -0,0 +1,31 @@
+FROM node AS build
+
+COPY node-zwave-js /root/node-zwave-js
+COPY zwavejs2mqtt /root/zwavejs2mqtt
+
+WORKDIR /root/node-zwave-js
+RUN npm install
+RUN npm run build
+
+WORKDIR /root/zwavejs2mqtt
+RUN npm config set unsafe-perm true
+RUN npm install
+RUN npm run build
+# RUN npm prune --production
+RUN cp -rL /root/node-zwave-js/packages/zwave-js node_modules/
+RUN rm -rf \
+    build \
+    index.html \
+    package-lock.json \
+    package.sh \
+    src \
+    static \
+    stylesheets
+
+FROM node
+LABEL maintainer="robertsLando"
+# Copy files from build stage
+COPY --from=build /root/zwavejs2mqtt /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
```suggestion
```

Remove this lines, you are creating a new step here

Reply:
This step is to get the size down, and reduce the number of layers. I prefer to keep it. I would prefer to get alpine working, for size, but it wasn't working with the latest code.

Reply:
Ok but if you use this you also have to copy the zwave-js folder